subsys/net/l2/ethernet/dsa/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 Denx Software Engineering GmbH.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig NET_DSA
+	bool "Enable Distributed Switch Architecture support (DSA)"
+	select NET_MGMT
+	select NET_MGMT_EVENT
+	help
+	  Enable Distributed Switch Architecture  support.
```

Comment:
Two spaces between "Architecture" and "support", one is enough

Reply:
Ok.